builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -345,15 +345,21 @@ node() {
 
               def envBucketKey = "${env_key}"
               echoServerlessFile()
+              def cfStackName = "${instance_prefix}-${config['domain']}-${config['service']}-${environment_logical_id}"
```

Comment:
Can we add comments to clarify what is being done here?

Reply:
Done